runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsConfig.java

```diff
@@ -107,4 +107,14 @@ default boolean showRespawnTimer()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		position = 6,
+		keyName = "dampenDeadNPCs",
+		name = "Dampen dead NPC menu entries",
+		description = "Dampens the color of dead NPCs in menu entries")
+	default boolean dampenDeadNPCs()
```

Comment:
Can this just be a recolor option?

Reply:
It could, but I think it would need a different colour for dead highlighted NPCs and dead non-highlighted NPCs. I don't mind either method personally.

Reply:
You should add tests for this.

Reply:
Decided to go with this method to heavily simplify the PR.